_posts/2021-01-21-cockpit-236.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 236
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-236
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 236.
+
+## fslist channels: Include properties of changed files
+
+The [fslist channel API](https://github.com/cockpit-project/cockpit/blob/master/doc/protocol.md#payload-fslist1) lists files in a directory, watches for further changes to files, and sends events when changes happen. These notifications now include additional properties about the file: the owner and group name, file size, and time of last modification.
+
+These changes were made with the goal of implementing an efficient file manager Cockpit page.
```

Comment:
```suggestion
These changes have been made with the intent to implement an efficient file manager page for Cockpit.
```

Further suggestions... Perhaps we should even drop "for Cockpit", given the context?

But I'm still not so happy with this sentence. It feels clunky. I wonder if @allisonkarlitskaya might have additional ideas about it?

Reply:
But this is still too strong. There is *no* promise that such a page will ever be built, and the cockpit team does not plan to do that either. It came from a contributor, and is just an idea.

 - "This feature is a requisite for building an efficient file manager page."
 - "This allows the efficient implementation of a future file manager page."
 - 
:thinking:

I'm also fine with dropping that sentence entirely, if we don't have a good way to convey this.

Reply:
Continued in another thread @ https://github.com/cockpit-project/cockpit-project.github.io/pull/377#discussion_r561707308 (at least for a suggestion, as it combines the sentences).